1-mandatory/2-people-i-know.js

```diff
@@ -438,7 +447,7 @@ test("3 - Powernet email addresses", () => {
   ]);
 });
 
-test.only("4 - friends with Stacie Villarreal as a colleague", () => {
```

Comment:
Don't forget to remove the <.only >  from the test after verifying it, as leaving it can cause issues with GitHub checks and prevent=> passing